### DISCUSSION TITLE
Missing Event start and end time from schema

### DISCUSSION THREAD

**Discussion by igMike-V** [EOT-Event-Organizer-s-Toolbox/eot-codebase#123] [Ideas]
We need to separate the date and times.  at the very least we should add "start time"/"end time".

there is also a possibility of a public start and end time and a separate venue start and end time to account for setup/teardown.

**Comment by 0916dhkim** [comment]
That's a good point, but currently, TJS organizers only record the start time of each event. It is often not easy to set the hard end time for in-person social events.
